src/masoniteorm/query/grammars/MSSQLGrammar.py

```diff
@@ -166,3 +166,13 @@ def table_insert_column_string(self):
 
     def value_string(self):
         return "'{value}'{separator}"
+
+    def truncate_table_string(self, foreign_keys=False):
+        if not foreign_keys:
+            return "TRUNCATE TABLE {table}"
+
+        return (
+            "ALTER TABLE {table} NOCHECK CONSTRAINT ALL;"
```

Comment:
This should return a list instead of a semi colon separated string of sql statements